services/openshiftremove/src/index.js

```diff
@@ -194,23 +194,4 @@ ${lastError}
 
 }
 
-const retryHandler = async (msg, error, retryCount, retryExpirationSecs) => {
-  const {
-    projectName,
-    branch,
-    pullrequestNumber,
-    type
-  } = JSON.parse(msg.content.toString())
-
-  const openshiftProject = ocsafety(`${projectName}-${branch || pullrequestNumber}`)
-
-  sendToLagoonLogs('warn', projectName, "", "task:remove-openshift:retry", {error: error, msg: JSON.parse(msg.content.toString()), retryCount: retryCount},
-`*[${projectName}]* remove \`${openshiftProject}\` ERROR:
-\`\`\`
-${error}
-\`\`\`
-Retrying in ${retryExpirationSecs} secs`
-  )
-}
-
-consumeTasks('remove-openshift', messageConsumer, retryHandler, deathHandler)
+consumeTasks('remove-openshift', messageConsumer, deathHandler)
```

Comment:
that will unfortunately not work, as the `consumeTasks` expects as the third parameter a function that is called during a retry. Now the `deathHandler` will be called on a retry which will send a full error to the notification system on the first retry.

I would suggest to leave the function `retryHandler` existing, just don't do anything in it and just run `return`